odtp/dashboard/main.py

```diff
@@ -46,5 +55,14 @@ def components():
 
 
 app.add_static_files("/static", "static")
+url = "https://auth.dev.swisscustodian.ch/auth/realms/odtp/protocol/openid-connect/certs"
+audience = "custodian"
```

Comment:
It would be better to have these two values defined in a config file instead to avoid having to touch the code when you deploy somewhere else